packages/polaris-viz/src/components/shared/SeriesIcon/SeriesIcon.tsx

```diff
@@ -1,30 +1,20 @@
-import type {Shape, Color} from '@shopify/polaris-viz-core';
-import {useTheme} from '@shopify/polaris-viz-core';
+import type {Shape, Color, LineStyle} from '@shopify/polaris-viz-core';
 
 import {LinePreview} from '../../LinePreview';
 import {SquareColorPreview} from '../../SquareColorPreview';
 
 interface Props {
   color: Color;
-  isComparison?: boolean;
   shape?: Shape;
+  lineStyle?: LineStyle;
 }
 
-export function SeriesIcon({
-  color,
-  isComparison = false,
-  shape = 'Bar',
-}: Props) {
-  const selectedTheme = useTheme();
-
+export function SeriesIcon({color, lineStyle, shape = 'Bar'}: Props) {
```

Comment:
Thought of refactoring this a bit, it didn't feel like SeriesIcon should concern about wether if its a comparison or not, I think we should pass in the `color` and `lineStyle` instead and calculate those from the parents